requirements.txt

```diff
@@ -1,71 +1,7 @@
-appnope==0.1.0
-backports-abc==0.5
-backports.shutil-get-terminal-size==1.0.0
-backports.ssl-match-hostname==3.5.0.1
-backports.weakref==1.0rc1
-bleach==3.1.4
-certifi==2017.7.27.1
-configparser==3.5.0
-cycler==0.10.0
-decorator==4.1.2
-entrypoints==0.2.3
-enum34==1.1.6
-funcsigs==1.0.2
-functools32==3.2.3.post2
-h5py==2.7.1
-html5lib==0.9999999
-ipykernel==4.6.1
-ipython==5.4.1
-ipython-genutils==0.2.0
-ipywidgets==7.0.0
-Jinja2==2.9.6
-jsonschema==2.6.0
-jupyter==1.0.0
-jupyter-client==5.2.0
-jupyter-console==5.2.0
-jupyter-core==4.4.0
-Keras==2.0.8
-Markdown==2.6.9
-MarkupSafe==1.0
-matplotlib==2.0.2
-mistune==0.8.1
-mock==2.0.0
-nbconvert==5.3.1
-nbformat==4.4.0
-notebook==5.7.8
-numpy==1.16
-pandas==0.20.3
-pandocfilters==1.4.2
-pathlib2==2.3.0
-pbr==3.1.1
-pexpect==4.2.1
-pickleshare==0.7.4
-prompt-toolkit==1.0.15
-protobuf==3.6.1
-ptyprocess==0.5.2
-Pygments==2.2.0
-pyparsing==2.2.0
-python-dateutil==2.6.1
-pytz==2017.2
-PyYAML==4.2b1
-pyzmq==17
-qtconsole==4.3.1
-scandir==1.5
-scikit-learn==0.19.0
-scipy==0.19.1
-seaborn==0.9.0
-Send2Trash==1.5.0
-simplegeneric==0.8.1
-singledispatch==3.4.0.3
-six==1.10.0
+matplotlib==2.2.5
+numpy==1.16.6
+pandas==0.24.2
+seaborn==0.9.1
 sklearn==0.0
-subprocess32==3.2.7
-tensorflow==1.14.0
-tensorboard==1.14.0
-terminado==0.8.1
-testpath==0.3.1
-tornado==4.5.2
-traitlets==4.3.2
-wcwidth==0.1.7
-Werkzeug==0.15.3
-widgetsnbextension==3.0.2
+tensorflow==1.14.0; python_version < '3.0'
+tensorflow==2.3.1; python_version > '3.0'
```

Comment:
Why not keep this at 1.15.4 for PY3? IMO easier to maintain this at TF1 for now.